CFC_DataCollector/trip_generator/trip_gen.py

```diff
@@ -96,13 +120,16 @@ def get_one_random_point_in_radius(address, radius):
     t = 2 * math.pi * v
     x = w * math.cos(t)
     y = w * math.sin(t)
+    print "type of y is %s" % type(y_0)
+    print "x = %s" % x
     x = float(x) / float(math.cos(y_0))   # To account for Earth something 
-    return Coordinate(x + x_0, y + y_0)
+    to_return = Coordinate(y + y_0, x + x_0)
+    print to_return
+    if abs(to_return.get_lat()) < 30 or abs(to_return.get_lon() < 30):
```

Comment:
Why do you need this? Add a comment in the file describing why?
What is so magical about 30, specially for longitude.